### ISSUE TITLE
Determine framework/pattern to use to test console scripts

### ISSUE THREAD

**Issue by stscieisenhamer** [spacetelescope/jwst#2904]
# Issue
Currently console scripts are not tested. Determine framework/pattern to use to test console scripts. Some options are:
- Simply use `subprocess` and check output.
- Use a `pytest` plugin, such as [`pytest-console-scripts`](https://github.com/kvas-it/pytest-console-scripts)
- ???

**Comment by jdavies-st** [comment]
Are you referring to `strun`, `stspec`, etc?

**Comment by stscieisenhamer** [comment]
Yep, anything defined as a `console-script` entry point and/or in the `scripts` folder.

**Comment by melanieclarke** [comment]
There are currently some minimal tests to check for installation of command-line scripts, in both `stpipe` and `jwst`, as well as some unit tests for specific behaviors in the scripts.  Test coverage could be better, but I don't see a strong need for dedicated effort to develop a test harness for these scripts at this point.  

I will close this stale ticket for lack of interest, but please reopen or file a new ticket if this is still desired.
